### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Vova-Stoliar**

### CODE REVIEW

**Comment by andrii-hryhoruk** [comment]
Nice solution)

**Review by andrii-hryhoruk — APPROVED** [review]

File: src/arrayMethodSort.js [hunk 1]
```diff
@@ -3,9 +3,44 @@
...
+  switch (true) {
+    case aToString > bToString:
+      return 1;
+
+    case aToString === bToString:
+      return 0;
+
+    default:
+      return -1;
+  }
```
Comment (VinnieJ-2k20) [inline src/arrayMethodSort.js:20]: This use of `switch/case` is valid, but not very appropriate. Rewrite using `if/return` structure.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Solution is correct, but let's fix some code style.

File: src/arrayMethodSort.js [hunk 2]
```diff
@@ -3,9 +3,41 @@
...
+
+const compareAsString = (a, b) => {
+  const aToString = String(a);
+  const bToString = String(b);
+
+  if (aToString > bToString) {
+    return 1;
+  } else if (aToString === bToString) {
+    return 0;
+  }
```
Comment (VinnieJ-2k20) [inline src/arrayMethodSort.js:15]: If you write `return` inside `if` , `else` is redundant.
```suggestion
  if (aToString > bToString) {
    return 1;
  } 
  
  if (aToString === bToString) {
    return 0;
  }
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Final fix to `if` structure.

**Review by VinnieJ-2k20 — APPROVED** [review]

**Pull request closed without merging**